Urban-Fun/Cells/TimelineCell.m

```diff
@@ -16,6 +17,9 @@ @implementation TimelineCell
 - (void)awakeFromNib {
     [super awakeFromNib];
     // Initialization code
+    self.shimmeringView.contentView = self.activityTitleLabel;
```

Comment:
Is good that you integrated the shimmering library to your app! I think usually we use the shimmering for items that are loading instead of items that are already present.
A good use for this would be to create a set of dummy cells with shimmering effect in the meantime you process your network request and then update the tableview with the results, for example in [this tutorial](https://youtu.be/Qq3Evspeea8?t=960)